evap/results/tools.py

```diff
@@ -451,7 +451,7 @@ def textanswers_visible_to(contribution):
     delegates = {delegate for contributor in non_proxy_contributors for delegate in contributor.delegates.all()}
     num_delegates = len(delegates - contributors)
 
-    sorted_contributors = sorted(contributors, key=lambda user: (user.last_name, user.first_name))
+    sorted_contributors = sorted(contributors, key=lambda user: (user.ordering_key))
```

Comment:
Maybe we shouldn't make it a property so that we can use `UserProfile.ordering_key` instead of a lambda here?